packages/overlay/test/restore-focus.test.js

```diff
@@ -116,6 +171,16 @@ describe('restore focus', () => {
         expect(isElementFocused(focusable)).to.be.true;
       });
 
+      it('should not restore focus-ring attribute if focus was moved outside overlay', async () => {
+        focusInput.focus();
+        focusInput.setAttribute('focus-ring', '');
```

Comment:
We might want to extract this to `focus-utils.js` at some point in the future. Not a blocker.